src/expressions.md

```diff
@@ -85,6 +84,33 @@ in the order given by their associativity.
 | `=` `+=` `-=` `*=` `/=` `%=` <br> `&=` <code>&#124;=</code> `^=` `<<=` `>>=` | right to left |
 | `return` `break` closures   |                     |
 
+## Evaluation order of operands
+
+Unless otherwise stated on the expression's page, evaluation of the operands of
```

Comment:
Can we cross-reference any cases where this *isn't* true from within this section, to not leave people wondering "what expressions *don't* evaluate their sub-expressions left-to-right"?

Reply:
I listed every case where it is true. I was being lazy with that sentence prior. I'm still being lazy by not actually linking to those expression pages, but that can always be done later and they're in the sidebar.

Reply:
One prominent case that is not mentioned here is `a = b`. Our execution odrer there, I *believe*, is to evaluate `b` before `a`, though I want to double check now =)

Reply:
[Yes](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=dedd696fe07142ecf4bb4a4c53d5496f)

Reply:
Hm, I seem to recall @eddyb saying that it actually is even more complicated than that, but I forgot the details...